src/react-apps/applications/receipt/package.json

```diff
@@ -0,0 +1,106 @@
+{
+  "name": "receipt-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "cross-env NODE_ENV=development webpack-dev-server --config webpack.config.development.js --mode development --open",
+    "build": "cross-env NODE_ENV=production webpack --config webpack.config.production.js --progress",
+    "build-develop": "cross-env NODE_ENV=development webpack --config webpack.config.development --progress --watch",
+    "test": "jest",
+    "compile-ts": "tsc",
+    "lint": "tslint -c tslint.json 'src/**/*.{ts,tsx}",
+    "clean": "rimraf dist compiled",
+    "format": "prettier-tslint fix src/**/*.{ts,tsx}",
+    "webpack-watch": "cross-env NODE_ENV=development webpack --config webpack.config.development.js --watch --progress",
+    "lintfix": "tslint -c tslint.json 'src/**/*.{ts,tsx} --fix"
+  },
+  "author": "Altinn",
+  "license": "3-Clause BSD",
+  "dependencies": {
+    "@babel/core": "7.4.4",
+    "@material-ui/core": "4.3.2",
+    "moment": "^2.24.0",
```

Comment:
Should use exact versions on depencencies.

Reply:
Fixed.